docs/Contributing/Advanced-Configuration.md

```diff
@@ -0,0 +1,378 @@
+Whether SMTP support is enabled or not to send emails from Fleet.
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  smtp_settings:
+    enable_smtp: true
+  ```
+
+### smtp_settings.enable_ssl_tls
+
+TODO(mna): doesn't seem to be used?
+Whether to enable SSL/TLS for the SMTP connection.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  smtp_settings:
+    enable_ssl_tls: false
+  ```
+
+### smtp_settings.enable_start_tls
+
+Whether to detect if TLS is used by the SMTP server and start using it if so.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  smtp_settings:
+    enable_start_tls: false
+  ```
+
+### smtp_settings.password
+
+> **Warning:** Be careful not to store your SMTP credentials in source control. It is recommended to set the password through the web UI or `fleetctl` and then remove the line from the checked in version. Fleet will leave the password as-is if the field is missing from the applied configuration.
+
+The password to use for the SMTP authentication, when `authentication_type` is set to `authtype_username_password`.
+
+- Optional setting (string).
+- Default value: "".
+- Config file format:
+  ```
+  smtp_settings:
+    password: supersekretsmtppass
+  ```
+
+### smtp_settings.port
+
+The port to use to connect to the SMTP server.
+
+- Optional setting (integer).
+- Default value: `587` (the standard SMTP port).
+- Config file format:
+  ```
+  smtp_settings:
+    port: 5870
+  ```
+
+### smtp_settings.sender_address
+
+The email address to use as sender for emails sent by Fleet.
+
+- Optional setting (string).
+- Default value: "".
+- Config file format:
+  ```
+  smtp_settings:
+    sender_address: fleet@example.org
+  ```
+
+### smtp_settings.server
+
+The server hostname for SMTP.
+
+- Optional setting, required to properly configue SMTP (string).
+- Default value: "".
+- Config file format:
+  ```
+  smtp_settings:
+    server: mail.example.org
+  ```
+
+### smtp_settings.user_name
+
+> **Warning:** Be careful not to store your SMTP credentials in source control. It is recommended to set the password through the web UI or `fleetctl` and then remove the line from the checked in version. Fleet will leave the password as-is if the field is missing from the applied configuration.
+
+The username to use for the SMTP authentication, when `authentication_type` is set to `authtype_username_password`.
+
+- Optional setting (string).
+- Default value: "".
+- Config file format:
+  ```
+  smtp_settings:
+    user_name: test_user
+  ```
+
+### smtp_settings.verify_ssl_certs
+
+Whether the SMTP server's SSL certificates should be verified. Can be turned off if self-signed certificates are used by the SMTP server.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  smtp_settings:
+    verify_ssl_certs: false
+  ```
+
+
+<meta name="pageOrderInSection" value="1100">
```

Comment:
This would put this page just before the "FAQ" in the "Contributing" section (and after the "Contributing" page).